src/guides/v2.4/test/static/static_test_execution.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
This file is identical to the 2.3 version - therefore it should be created as a symlink, not a separate file.